website/docs/advanced/builder.md

```diff
@@ -0,0 +1,190 @@
+---
+id: builder
+title: Configure MEV Builder
+sidebar_label: Configure MEV Builder
+---
+
+import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';
+
+<HeaderBadgesWidget commaDelimitedContributors="James" lastVerifiedDateString="May 18th, 2023" lastVerifiedVersionString="v4.0.2" />
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import BuilderPng from '@site/static/img/builder.png';
+
+:::caution
+
+This guide is for advanced Prysm users to configure their client for the purposes of extracting [MEV](https://ethereum.org/en/developers/docs/mev/). 
+There are risks to using a builder which may result in missed rewards, missed proposals, censored transactions, omitted transactions, and even slashing when used incorrectly. The Prysm team does not provide guidance on which builders/relays are recommended but lists are available for you to make a judgement based on your own values. 
+
+:::
+
+The provided guide offers explanations on configuring the Prysm client to utilize a [custom builder](https://docs.flashbots.net/flashbots-mev-boost/block-builders) through a [relay](https://docs.flashbots.net/flashbots-mev-boost/relay). The relay acts as a middleware that connects validators to block builders. This configuration involves adjusting flags and settings on both the validator client and the beacon node. It's important to note that this guide does not cover setting up your own relay, builder, or MEV-boost (a middleware aggregator of relays). For detailed instructions on those components, please refer to their respective guides for up-to-date information. In the Prysm client, the builder is used through the relay to produce execution payloads containing transactions that maximize the validator's benefits, prioritizing them over the execution client. However, the execution client will still be necessary for Prysm to handle synchronization and serve as a fallback option in case any issues arise while utilizing the builder. The builder will only come into play when there is a validator proposal.
```

Comment:
This paragraph is very long when looking at the deploy preview. It would be good to probably put the caveats into an info box and make it shorter. Here's an idea

```suggestion
The provided guide offers explanations on configuring the Prysm client to utilize a [custom builder](https://docs.flashbots.net/flashbots-mev-boost/block-builders) through a [relay](https://docs.flashbots.net/flashbots-mev-boost/relay). The relay acts as a middleware that connects validators to block builders. This configuration involves both the validator client and the beacon node. It's important to note this guide does not cover setting up your own relay, builder, or MEV-boost software. 

:::info
In the Prysm client, the builder is used through the relay to get transactions that maximize the validator's benefits, prioritizing them over local ones. However, the execution client will still be necessary as a fallback option in case any issues arise while utilizing the builder. The builder will only come into play when there is a validator proposal.
:::
```